prov/usnic/src/usdf_ep_rdm.c

```diff
@@ -369,13 +338,8 @@ int usdf_rdm_fill_rx_attr(uint32_t version, struct fi_info *hints,
 		return -FI_ENODATA;
 
 	/* clear the mode bits the app doesn't support */
-	defaults.mode &= (hints->mode | hints->rx_attr->mode);
-
-	/* In version < 1.5, FI_LOCAL_MR is required. */
-	if (FI_VERSION_LT(version, FI_VERSION(1, 5))) {
-		if ((defaults.mode & FI_LOCAL_MR) == 0)
-			return -FI_ENODATA;
-	}
+	if (hints->mode || hints->rx_attr->mode)
```

Comment:
> I think you are confusing this mode with mr_mode bit.
> This one is simply mode from fi_info. It can be 0, and if it is 0, we return the defaults.

Oh right, my bad.